### DISCUSSION TITLE
US Libre 3 and reader help

### DISCUSSION THREAD

**Discussion by Kustomwireless** [maheini/FreeStyle-Libre-3-patch#58] [General]
Can a US  Libre 3 sensor be started with the Libre 3 reader then be read with Juggluco and data passed to Xdrip???

**Comment by maheini** [comment]
Hi @Kustomwireless 
I haven't tested it myself, but based on reportings from other users, it should be possible with US-sensors (tested working with EU sensors). Please pay attention here:

- Log in to your librelink account before starting the sensor.
- Use the same account for Juggluco
- Ensure you are logged in in Juggluco
- Follow the instructions for more infos.

If you got any questions or one does know more about this, please feel free to comment.

**Comment by j-kaltes** [comment]
If the Libre 3 reader isn't connected to a Libreview Account while starting the sensor, it isn't possible to take the sensor to any other app or device. During the start of the sensor a number is send to the sensor and this number has to be the same the next time the sensor is scanned with NFC. In Abbott's Libre 3 app this number is received from Libreview. The Libre 3 reader that isn't connected to a Libreview account will use another unknown number.

**Comment by markd89** [comment]
@maheini thanks for your great app!

I am like the OP. I have the Libre3 reader which is a stand-alone device to read the sensor.  It activates the sensor and reads data but is not internet connected. I am in the USA.

Is there any way to use your app in my situation?

Can I start the sensor with Juggluco and not use the reader or the Libre3 app?

(I have a rooted LineageOS phone and the Libre3 Android app starts and quits so I can't use it.)
